src/main/java/dev/dbos/transact/workflow/StepInfo.java

```diff
@@ -1,14 +1,21 @@
 package dev.dbos.transact.workflow;
 
 public class StepInfo {
+  public static record StepInfoError(
+      String className, String message, String serializedError, Throwable throwable) {}
```

Comment:
this is the same as WorkflowStatusError? Mabe we just have an ExecptionInfo record shared between both?